src/FormBuilder/resources/js/form-builder/src/containers/BlockEditorInterfaceSkeletonContainer.tsx

```diff
@@ -42,7 +44,11 @@ const DesignEditorSkeleton = () => {
                     showSidebar={showSidebar}
                     toggleShowSidebar={toggleShowSidebar}
                     onSaveNotice={() => {
```

Comment:
Can we extract an `onSaveNoticeCallback` function to be shared by the two components? We recently duplicated the interface skeleton to account for design mode, but it could use a refactor for the props that are shared. As a start I think we can extract the notice logic to a shared function.

Reply:
@kjohnson the problem with shared function is that we have to use `createSuccessNotice` from `useDispatch('core/notices')` and that has to be inside the functional component. Maybe a better option would be to move all that logic inside the `HeaderContainer` component and remove the `onSaveNotice` callback.

Reply:
I just did that https://github.com/impress-org/givewp/pull/6986/commits/d4d41e8850375b48a8814a3933f9c12281ffc87e

Reply:
Cool. That sounds good!